Create Blog Post “texas-attorney-general’s-sites-for-death-and-shooting-reports-go-dark” (#562)
texas-justice-initiative/website-nextjs: content/blog/posts/texas-attorney-general’s-sites-for-death-and-shooting-reports-go-dark.md

---
title: "Texas Attorney General’s Sites for Death and Shooting Reports Go Dark "
subtitle: ""
date: 2021-03-20
authors:
  - Eva Ruth Moravec
---

> For five weeks, the Texas Office of the Attorney General’s websites that house reports on [officer-involved shootings](https://oagtx.force.com/oisreports/apex/OISReportsPage) and [deaths in custody](https://oagtx.force.com/cdr/cdrreportdeaths) have been malfunctioning, restricting access to records that detail important interactions between civilians and law enforcement. 

Reviewer: ```suggestion
For five weeks, the Texas Office of the Attorney General’s websites that house reports on [officer-involved shootings](https://oagtx.force.com/oisreports/apex/OISReportsPage) and [deaths in custody](https://oagtx.force.com/cdr/cdrreportdeaths) have been malfunctioning, cutting off access to records that detail important (and often lethal) interactions between civilians and law enforcement. 
```

Reviewer: I think a stronger word than restricting is needed because to me that suggests that some people still have access to the info when in reality no one (outside of the agency) does

Author: thanks - great suggestion!

> For five weeks, the Texas Office of the Attorney General’s websites that house reports on [officer-involved shootings](https://oagtx.force.com/oisreports/apex/OISReportsPage) and [deaths in custody](https://oagtx.force.com/cdr/cdrreportdeaths) have been malfunctioning, cutting off access to records that detail important (and often lethal) interactions between civilians and law enforcement. 

Author: ```suggestion
For six weeks, the Texas Office of the Attorney General’s websites that house reports on [officer-involved shootings](https://oagtx.force.com/oisreports/apex/OISReportsPage) and [deaths in custody](https://oagtx.force.com/cdr/cdrreportdeaths) was malfunctioning, cutting off access to records that detail important (and often lethal) interactions between civilians and law enforcement. 
```

> Every week, law enforcement agencies file dozens of reports with the Attorney General’s office that give people a glimpse at incidents like an officer’s use of deadly force, an incarcerated individual’s death and shootings in which an officer is shot or shoots a civilian, resulting in an injury or a death. The office then posts these reports online – except that part of the process is currently broken. 

Reviewer: ```suggestion
Every week, law enforcement agencies file dozens of reports with the Attorney General’s office that provide a glimpse into incidents like an officer’s use of deadly force, an incarcerated individual’s death, and shootings in which an officer is shot or shoots a civilian, resulting in an injury or a death. The office then posts these reports online – except since the weather emergency when all updates ceased. 
```

Reviewer: I promise I only added the comma in because I think it clarifies the sentence!

> Every week, law enforcement agencies file dozens of reports with the Attorney General’s office that provide a glimpse into incidents like an officer’s use of deadly force, an incarcerated individual’s death, and shootings in which an officer is shot or shoots a civilian, resulting in an injury or a death. The office then posts these reports online – except since the weather emergency when all updates ceased.  

Author: ```suggestion
Every week, law enforcement agencies file dozens of reports with the Attorney General’s office that provide a glimpse into incidents like an officer’s use of deadly force, an incarcerated individual’s death, and shootings in which an officer is shot or shoots a civilian, resulting in an injury or a death. The office then posts these reports online – except that hasn't happened since severe winter weather hit the state and all updates ceased.  
```

> Every week, law enforcement agencies file dozens of reports with the Attorney General’s office that provide a glimpse into incidents like an officer’s use of deadly force, an incarcerated individual’s death, and shootings in which an officer is shot or shoots a civilian, resulting in an injury or a death. The office then posts these reports online – except that hasn't happened since severe winter weather hit the state and all updates ceased.  

Author: ```suggestion
Every week, law enforcement agencies file dozens of reports with the Attorney General’s office that provide a glimpse into incidents like an officer’s use of deadly force, an incarcerated individual’s death, and shootings in which an officer is shot or shoots a civilian, resulting in an injury or a death. The office then posts these reports online – except that didn't happen for several weeks after severe winter weather hit the state and all updates ceased.  
```

> On Feb. 15, I noticed that both repositories were empty. After a couple of weeks, some reports appeared, but both collections are still missing all reports filed in 2021 and nearly all filed in 2020, too. 

Reviewer: ```suggestion
On Feb. 15, I noticed that both repositories were empty. After a couple of weeks, some reports appeared, but both collections are still missing all reports filed in 2021 and nearly all the ones that had been filed in 2020, as well. 
```

> On Feb. 15, I noticed that both repositories were empty. After a couple of weeks, some reports appeared, but both collections are still missing all reports filed in 2021 and nearly all the ones that had been filed in 2020, as well. 

Author: ```suggestion
On Feb. 15, I noticed that both repositories were empty. After a couple of weeks, some reports appeared, but both collections were still missing all reports filed in 2021 and nearly all the ones that had been filed in 2020, as well.  The last week of March, all reports were seemingly restored and the issue appeared to be resolved. 
```

> State laws require agencies to file these reports with the Texas Attorney General within 30 days of an incident in which an [officer is shot or shoots someone](https://statutes.capitol.texas.gov/Docs/CR/htm/CR.2.htm#2.139), or when [someone dies in their custody](https://statutes.capitol.texas.gov/Docs/CR/htm/CR.49.htm). The office then posts these reports online, which – for the officer-involved shooting reports – is a move stipulated in statute. 

Reviewer: ```suggestion
State laws require agencies to file these reports with the Texas Attorney General within 30 days of an incident in which an [officer is shot or shoots someone](https://statutes.capitol.texas.gov/Docs/CR/htm/CR.2.htm#2.139), or when [someone dies in their custody](https://statutes.capitol.texas.gov/Docs/CR/htm/CR.49.htm). The office then posts these reports online – a move stipulated by statute for the officer-involved shooting reports. 
```

> Said another way: By not publishing the reports on officer-involved shootings, the OAG is violating state law. 

Reviewer: love this sentence

Reviewer 2: Love it too!

> Said another way: By not publishing the reports on officer-involved shootings, the OAG is violating state law. 

Author: ```suggestion
Said another way: By not publishing the reports on officer-involved shootings, the OAG was violating state law. 
```

Back when the sites went down, I sent an email informing the office as such, as I have done in the past. On Feb. 22, I heard back:

> Hi Ms. Moravec,
>
> The OAG was closed all last week due to the weather and the electrical outages.  I have reported the situation to IT this morning and they let me know they are working on it.
>
> As soon as I hear back from them, I will let you know.  Thank you for reporting this to us.

> IT then followed up that “Salesforce recently rolled out another update,” which caused the outage. About a week later, many of the cases had been restored, but hundreds from this year and last year remain missing, which I reported to the Attorney General’s office. 

Reviewer: ```suggestion
IT then followed up that “Salesforce recently rolled out another update,” which caused the outage. About a week later, many of the cases had been restored, but hundreds from this year and last year remain missing, which I again reported to the Attorney General’s office. 
```

> This outage means that when someone dies in law enforcement custody – such as Marvin Scott III, 26, who died in the Collin County Jail in March, the custodial death report that officials must file related to his death isn't immediately visible online. Reports (once they are filed) and data are still obtainable through open records requests. 

Reviewer: ```suggestion
This outage means that when someone dies in law enforcement custody – such as Marvin Scott III, 26, who died in the Collin County Jail in March, the custodial death report that officials must file related to the death isn't visible online. That being said, reports (once they are filed) and data are still obtainable through open records requests. 
```

Reviewer: I think it is important to take out the "immediately" since the turn around time isn't that fast usually as they milk the 30 day allowance

> This outage means that when someone dies in law enforcement custody – such as Marvin Scott III, 26, who died in the Collin County Jail in March, the custodial death report that officials must file related to the death isn't visible online. That being said, reports (once they are filed) and data are still obtainable through open records requests. 

Author: ```suggestion
This outage means that when someone dies in law enforcement custody – such as Marvin Scott III, 26, who died in the Collin County Jail in March, the custodial death report that officials must file related to the death isn't visible online. That being said, reports (once they are filed) and data are still obtainable through open records requests. On March 22, we submitted an open records request to the Attorney General's office for the custodial death report for Marvin Scott III – not even knowing if it had indeed been filed. We received the report the next day and posted it publicly [here](https://www.documentcloud.org/documents/20521178-marvinscottiii_cdr). 
```

Author: just cot his custodial death report today.

> This outage means that when someone dies in law enforcement custody – such as Marvin Scott III, 26, who died in the Collin County Jail in March, the custodial death report that officials must file related to the death isn't visible online. That being said, reports (once they are filed) and data are still obtainable through open records requests. 

Author: ```suggestion
This outage meant that when someone dies in law enforcement custody – such as Marvin Scott III, 26, who died in the Collin County Jail in March – the custodial death report that officials must file related to the death wasn't visible online. That being said, reports (once they are filed) and data were still obtainable through open records requests. 
```

> While the sites have been malfunctioning, TJI has continued to obtain data on these incidents from the Attorney General’s office through our standard monthly open records requests. While the data on our site reflects reports filed through March 1, the actual reports are still unavailable to the public. 

Author: ```suggestion
While the sites were malfunctioning, TJI continued to obtain data on these incidents from the Attorney General’s office through our standard monthly open records requests. We also stayed in touch with the office, which informed us March 29 that the sites had been fully restored. 
```

Reviewer 2: Do we know whether the OAG website has any records missing (during the outage) at this point or whether they fully fixed the problem?

> This lag in transparency has taken place during a tenuous time in Texas, with the novel coronavirus still a threat, particularly to those who are incarcerated, and also amid winter storms that knocked out power for millions of Texans [and claimed the lives of at least 57](https://www.texastribune.org/2021/03/15/texas-winter-storm-deaths/). It remains unclear exactly how the storm affected the health of those behind bars, but we promise to keep apprised of this situation and follow up.

Reviewer: ```suggestion
This lag in transparency has taken place during a tenuous time in Texas, with the novel coronavirus still a threat, particularly to those who are incarcerated, and amid winter storms that knocked out power for millions of Texans [and claimed the lives of at least 57 people](https://www.texastribune.org/2021/03/15/texas-winter-storm-deaths/). It remains unclear exactly how the storm affected the health of those behind bars, but we promise to keep apprised of this situation and always share what we find.
```

> This lag in transparency has taken place during a tenuous time in Texas, with the novel coronavirus still a threat, particularly to those who are incarcerated, and amid winter storms that knocked out power for millions of Texans [and claimed the lives of at least 57 people](https://www.texastribune.org/2021/03/15/texas-winter-storm-deaths/). It remains unclear exactly how the storm affected the health of those behind bars, but we promise to keep apprised of this situation and always share what we find.

Author: ```suggestion
This lag in transparency took place during a tenuous time in Texas, with the novel coronavirus still a threat, particularly to those who are incarcerated, and amid winter storms that knocked out power for millions of Texans [and claimed the lives of at least 57 people](https://www.texastribune.org/2021/03/15/texas-winter-storm-deaths/). It remains unclear exactly how the storm affected the health of those behind bars, but we promise to keep apprised of this situation and always share what we find.
```
